src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -1357,6 +1366,14 @@ private uint getClassAttribsInternal(TypeDesc type)
                     result |= CorInfoFlag.CORINFO_FLG_ABSTRACT;
             }
 
+#if READYTORUN
+            if (!_compilation.NodeFactory.CompilationModuleGroup.VersionsWithType(type))
```

Comment:
Can this be `_compilation.CompilationModuleGroup` like above?

Reply:
Good point, fixed in 3rd commit (including one other place in the file).